probe/plugins/unix_round_tripper.go

```diff
@@ -0,0 +1,27 @@
+package plugins
+
+import (
+	"net"
+	"net/http"
+	"net/http/httputil"
+	"time"
+)
+
+type unixRoundTripper struct {
+	address string
+	timeout time.Duration
+}
+
+func makeUnixRoundTripper(address string, timeout time.Duration) (http.RoundTripper, error) {
+	return unixRoundTripper{address: address, timeout: timeout}, nil
+}
+
+func (t unixRoundTripper) RoundTrip(req *http.Request) (*http.Response, error) {
+	conn, err := net.DialTimeout("unix", t.address, t.timeout)
+	if err != nil {
+		return nil, err
+	}
+	client := httputil.NewClientConn(conn, nil)
```

Comment:
After hitting my head against the wall for hours debugging a problem in the python plugin, it turns out that the `httputil` client was the culprit.

This commit removes it, having the nice side effect https://github.com/weaveworks/scope/commit/62d5b2530792e55a2afcddb611bbfe94479db0d3 of reusing connections and dealing with `Connection: close` transparently.